istioctl/cmd/add-to-mesh.go

```diff
@@ -77,7 +77,7 @@ func addToMeshCmd() *cobra.Command {
 		Use:     "add-to-mesh",
 		Aliases: []string{"add"},
 		Short:   "Add workloads into Istio service mesh",
-		Long: `'istioctl experimental add-to-mesh' restarts pods with an Istio sidecar or configures meshed pod access to external services.
+		Long: `'Istioctl experimental add-to-mesh' restarts pods with an Istio sidecar or configures meshed pod access to external services.
```

Comment:
`istioctl` is a binary, it should not be capitalized.

Reply:
@howardjohn 
Oh. thanks for review.
Updated